rules.md

```diff
@@ -540,9 +540,10 @@ If the field is not present, then the condition will not match.
 The comparison is case-sensitive.
 The field can also include a prefix that changes the span used for evaluation of the field.
 The only prefix currently supported is `root`, as in `root.http.status`.
-Specifying `root.`  causes the condition to be evaluated against the root span.
-For example, if the `Field` is `root.url`, then the condition will be processed using the url  field from the root span.
-The setting `Scope: span` for a rule does not change the meaning of this prefix -- the condition is still evaluated on the root span and is treated as if it were part of the   span being processed.
+Specifying `root.` causes the condition to be evaluated against the root span.
+For example, if the `Field` is `root.url`, then the condition will be processed using the url field from the root span.
+The setting `Scope: span` for a rule does not change the meaning of this prefix -- the condition is still evaluated on the root span and is treated as if it were part of the  span being processed.
+When using the `not-exists` operator to check a condition with a `root.` prefix, if the root span does not exist on the trace then the condition will evaluate to false.
```

Comment:
I don't think this is true, or should be reworded.

If the root span does not exist, then the `exists` variable will be false, so `not-exists` will be true.

```suggestion
When using the `exists` or `not-exists` operators to check a field containing a `root.` prefix, if the trace has no root span, it will be treated as if the field did not exist.
```